compiler/daml-lf-tools/src/DA/Daml/LF/Simplifier.hs

```diff
@@ -115,6 +115,7 @@ safetyStep = \case
       BEGreaterEq _       -> Safe 2
       BEGreater _         -> Safe 2
       BEToText _          -> Safe 1
+      BECodePointsToText  -> Safe 1
```

Comment:
Since `CODE_POINTS_TO_TEXT` will throw an exception if there's an invalid code point, this should be
```suggestion
      BECodePointsToText  -> Safe 0
```